android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -417,7 +417,7 @@ private void downloadPageId() {
         }
     }
 
-    private void setupConnection() {
+    public void setupConnection() {
```

Comment:
How about renaming this method to reflect that this means dowloading data? 😄

Reply:
Yes that would be best. On it.